app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/settings/accountsecurity/loginapproval/LoginApprovalViewModel.kt

```diff
@@ -207,11 +207,17 @@ class LoginApprovalViewModel @Inject constructor(
     }
 
     private fun closeScreen() {
-        if (state.specialCircumstance?.shouldFinishWhenComplete == true) {
-            sendEvent(LoginApprovalEvent.ExitApp)
+        sendEventBasedOnSpecialCircumstance()
+    }
+
+    private fun sendEventBasedOnSpecialCircumstance() {
```

Comment:
Should this be `sendClosingEvent...` or something like that?

Reply:
Sounds good to me, I was trying to capture what was going on and why but I think "closing even" captures the idea of either case!